Dockerfile

```
@@ -112,7 +112,11 @@ RUN patch -d / -p0 < /tmp/bosco_cluster_xtrace.patch
 # HACK: Don't copy over the SSH pub key to the remote side. We set
 # this up with the site out of band.
 COPY hosted-ce/overrides/skip_key_copy.patch /tmp
-RUN patch -d / -p0 < /tmp/skip_key_copy.patch
+RUN [[ $BASE_YUM_REPO == "development" ]] || patch -d / -p0 < /tmp/skip_key_copy.patch
+RUN if ! fgrep -- '--copy-ssh-key' /usr/bin/bosco_cluster; then  \
```

Same thing here; if you don't want the matching line printed on stdout, you can add `-q`

```suggestion
RUN if ! fgrep -q -- '--copy-ssh-key' /usr/bin/bosco_cluster; then  \
```